website/models.py

```diff
@@ -312,6 +313,23 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def get_cve_score(self):
+        if self.cve_id is None:
+            return None
+        try:
+            url = "https://services.nvd.nist.gov/rest/json/cves/2.0?cveId=%s" % (self.cve_id)
+            response = requests.get(url).json()
+            results = response["resultsPerPage"]
+            if results != 0:
+                metrics = response["vulnerabilities"][0]["cve"]["metrics"]
+                if metrics:
+                    cvss_metric_v = next(iter(metrics))
+                    score = metrics[cvss_metric_v][0]["cvssData"]["baseScore"]
+                    return score
```

Comment:
Optional: another optimization you could use is 
```suggestion
                    return metrics[cvss_metric_v][0]["cvssData"]["baseScore"]
```
right away as you're using `score` variable just once.

Reply:
ohh yess thanks